If `logs` directory already exists, `make deps` exits with error status

This is a development environment only issue. If `logs` directory already exists, `make deps` fails with the following error:

```
mkdir logs
mkdir: logs: File exists
make: *** [deps] Error 1
```

This issue has no serious impact on developers because the dependencies are installed correctly prior to this error. In the interest of neatness, we should fix this. A simple solution is to use the [`mkdir -p` option](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/mkdir.html#tag_20_79_04), e.g., `mkdir -p logs` which does not report error for existing directories.

--- a/Makefile
+++ b/Makefile
@@ -21,7 +21,7 @@ deps: FORCE
 	touch venv
 	. ./venv && pip3 install -r requirements.txt
 	. ./venv && pip3 install -r dev-requirements.txt
-	mkdir logs
+	mkdir -p logs
 
 rmvenv: FORCE
 	rm -rf ~/.venv/cloudmarker

